source/agora/utils/InetUtils.d

```diff
@@ -86,7 +107,7 @@ struct InetUtils
                 ips ~= ip;
             }
         }
-
+        ips ~= publicAddresses();
```

Comment:
This should be a fallback, as there's no need to do this kind of query unless we didn't get something usable from getifaddr.

Reply:
Also we probably want to report that situation to the user.

Reply:
Will think of it.

Reply:
Modified.